lib/active_merchant/billing/gateways/quickbooks.rb

```diff
@@ -267,6 +269,10 @@ def extract_response_body_or_raise(response_error)
         end
         response_error.response.body
       end
+
+      def refund_uri(authorization)
```

Comment:
only that  ``` capture ``` alone uses this uri but the refund uri is to be shared with voids, which is coming later. I can factor this out then too